railties/test/commands/middleware_test.rb

```diff
@@ -0,0 +1,427 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rails/command"
+require "rails/commands/middleware/middleware_command"
```

Comment:
I think we can drop this `require`.

```suggestion
```

Reply:
Thanks! I've removed it.
This pattern might be used in other tests as well.
I'll see if I can remove some more requires in another PR.